packages/eslint-plugin-pf-codemods/lib/rules/v5/warn-key-codes-removed.js

```diff
@@ -0,0 +1,23 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8174
+module.exports = {
+  meta: { fixable: "code" },
```

Comment:
should this be here if we aren't doing a code fix? also, did you mean this rule to add a warning vs error?

Reply:
Hmmm, probably not? I'll try removing and see what happens.

And yes, the intent was for it to be a warning. If we want it to be an error though I'm fine with that.